starboard/android/shared/media_is_video_supported.cc

```diff
@@ -92,6 +92,12 @@ bool SbMediaIsVideoSupported(SbMediaVideoCodec video_codec,
                                      false)) {
       MaxMediaCodecOutputBuffersLookupTable::GetInstance()->SetEnabled(false);
     }
+
+    // Allow the web app to set the maximum size of input buffer
+    // (MediaFormat.KEY_MAX_INPUT_SIZE)
+    if (!mime_type->ValidateIntParameter("maxinputsize")) {
```

Comment:
If we are going to expose this via dom interface, consider removing `maxinputsize` support as an mime attribute.

Reply:
Removed MIME attribute.